security/cmd/node_agent/management/server.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2018 Istio Authors
```

Comment:
For this PR it's fine. But going forward, i would prefer not to rename/move a file and adding content in the same time. I would be hard to track the actual logic change for reviewers and for tracking purpose :)

Reply:
Good point, will follow that in future.

Reply:
Thanks, github code review interface is not as good as critique :)